raiden/tests/utils/smoketest.py

```diff
@@ -311,10 +321,19 @@ def setup_raiden(
         registry_address=contract_addresses[CONTRACT_TOKEN_NETWORK_REGISTRY],
         contract_manager=contract_manager,
     )
-    registry.add_token(
-        token_address=to_canonical_address(token.contract.address),
-        given_block_identifier='latest',
-    )
+
+    if contracts_version == DEVELOPMENT_CONTRACT_VERSION:
```

Comment:
So the smoketest will now run for both development and production? Where is this configured?

Reply:
You could run `raiden --environment-type production smoketest`... but this isn't the point. This was added to satisfy CLI tests

Reply:
I don't get the response here. It seems that now with your change it's possible to run the smoke test for both production and development. That's good.

Can you add it to the CI?

Reply:
Sorry... Here are the CLI tests i was referring to:
https://github.com/raiden-network/raiden/blob/master/raiden/tests/integration/cli/test_cli_development.py

and 

https://github.com/raiden-network/raiden/blob/master/raiden/tests/integration/cli/test_cli_production.py

They both invoke raiden's interface where i noticed i have to check for the environment in this specific block.

> Can you add it to the CI?

Why do you think this is necessary?

Reply:
> Why do you think this is necessary?

Because we now have tests for both versions and both versions are easily supported in the CI. 

The smoke test is the first line of defense against bugs when we develop, the first thing we run along with the unit tests. We should run it with both the development and the red eyes version to see if we broke something with either version of the contracts.

Reply:
Done